docs/contributing/gatsby-style-guide.md

```diff
@@ -126,19 +126,45 @@ Hyperlinks should contain the clearest words to indicate where the link will lea
 ```markdown
 <!-- Good -->
 
-[Gatsby's docs](https://www.gatsbyjs.org/docs/)
+[Gatsby's docs](/docs/)
```

Comment:
I'd remove this change to the example. While this uses .org and links should be relative, the example is about hyperlinks themselves and the text that accompanies them. By removing the full URL it's harder to tell that it is a hyperlink.

Reply:
agree, longer url is more clear, 
i suggest link to https://www.gatsbyjs.com/cloud/

Reply:
changed to a full link, if you like more the https://www.gatsbyjs.org/docs/ then i can revert to it.

Reply:
I think that is a great choice!